### PR TITLE
Add apache httpd metrics crawler plugin

### PR DESCRIPTION
**Pull request by hitomitak**
This PR supports to gather apache httpd server provided metrics.
The detail information is described at issue #238

### CODE REVIEW

**Comment by sahilsuneja1** [comment]
@hitomitak; Thanks! Can you please address travis' flake's max_complexity error in `apache_crawler.py:retrieve_metrics()` ?

File: crawler/plugins/applications/apache/apache_crawler.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        "BytesPerReq": "BytesPerReq",
+        "ReqPerSec": "ReqPerSec",
+        "Uptime": "Uptime"
+    }
+
+    stats = {}
+
+    line_num = 0
+    for line in status:
+        line_num += 1
```
Comment (sahilsuneja1) [inline crawler/plugins/applications/apache/apache_crawler.py]: Is this being used somewhere?

**Comment by sahilsuneja1** [comment]
Thanks @hitomitak for the plugin, the test cases, and the fix! A small nit in the code.

File: crawler/plugins/applications/apache/apache_crawler.py (hunk 1 above)
Comment (hitomitak) [inline crawler/plugins/applications/apache/apache_crawler.py]: line_num is not used anywhere. Thank you for useful comments.I will remove this variable when I create a new PR to add a new application plugin to monitor tomcat.

**Pull request merged**